src/test/java/org/tron/common/runtime/vm/DepositTest.java

```diff
@@ -0,0 +1,276 @@
+    String bCode = "608060405234801561001057600080fd5b5061029f806100206000396000f3006080604052600436106100325763ffffffff60e060020a600035041663417c7cc581146100375780637a9a0e721461006a575b600080fd5b34801561004357600080fd5b5061006873ffffffffffffffffffffffffffffffffffffffff6004351660243561009b565b005b34801561007657600080fd5b5061006873ffffffffffffffffffffffffffffffffffffffff60043516602435610187565b60408051808201825260108082527f6368616e67654e2875696e74323536290000000000000000000000000000000060208301908152925191926000928492909182918083835b602083106101015780518252601f1990920191602091820191016100e2565b6001836020036101000a038019825116818451168082178552505050505050905001915050604051809103902090508373ffffffffffffffffffffffffffffffffffffffff168160e060020a9004846040518263ffffffff1660e060020a028152600401808281526020019150506000604051808303816000875af15050505050505050565b60408051808201825260108082527f6368616e67654e2875696e74323536290000000000000000000000000000000060208301908152925191926000928492909182918083835b602083106101ed5780518252601f1990920191602091820191016101ce565b6001836020036101000a038019825116818451168082178552505050505050905001915050604051809103902090508373ffffffffffffffffffffffffffffffffffffffff168160e060020a9004846040518263ffffffff1660e060020a028152600401808281526020019150506000604051808303816000875af19250505050600080fd00a165627a7a723058209cb73b547b599b5d2a96ade83e03d656f8025f4ac7642f8b9aab37cc2574f8310029";
+
+    long value = 0;
+    long fee = 100000000;
+    long consumeUserResourcePercent = 0;
+
+    Transaction aTrx = TVMTestUtils.generateDeploySmartContractAndGetTransaction(
+        contractA, address, aABI, aCode, value, fee, consumeUserResourcePercent, null);
+    Runtime runtime = TVMTestUtils.processTransactionAndReturnRuntime(aTrx, DepositImpl.createRoot(manager), null);
+    Assert.assertNull(runtime.getRuntimeError());
+
+    Transaction bTrx = TVMTestUtils.generateDeploySmartContractAndGetTransaction(
+        contractB, address, bABI, bCode, value, fee, consumeUserResourcePercent, null);
+    runtime = TVMTestUtils.processTransactionAndReturnRuntime(bTrx, DepositImpl.createRoot(manager), null);
+    Assert.assertNull(runtime.getRuntimeError());
+
+    byte[] aAddress = Wallet.generateContractAddress(aTrx);
+    byte[] bAddress = Wallet.generateContractAddress(bTrx);
+
+    Thread.sleep(10000);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this use of "Thread.sleep()". ([squid:S2925](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS2925))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1664&issues=AWbJU6rUSgN4uCclo1hO&open=AWbJU6rUSgN4uCclo1hO)

[ISSUE_KEY]: AWbJU6rUSgN4uCclo1hO